From 0807a294b30949f0bc311b6455a817332e6f1cfc Mon Sep 17 00:00:00 2001
From: Bayside308 <nhall0312@gmail.com>
Date: Mon, 15 May 2017 16:19:58 -0400
Subject: [PATCH 67/71] Update _DiwUtils.java

I added annotations for some of the variables and added a potential new number system.
---
 rainbow/src/main/java/org/projectrainbow/_DiwUtils.java | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/rainbow/src/main/java/org/projectrainbow/_DiwUtils.java b/rainbow/src/main/java/org/projectrainbow/_DiwUtils.java
index 9c0eec8..64dbf6a 100644
--- a/rainbow/src/main/java/org/projectrainbow/_DiwUtils.java
+++ b/rainbow/src/main/java/org/projectrainbow/_DiwUtils.java
@@ -117,8 +117,8 @@ public class _DiwUtils {
     public static int JanitorInterval = 900;
     public static int JanitorWarnSecs = 30;
     public static String MC_VERSION_STRING;
-    public static String DiwModVersion = "112.1";
-    public static String Version = "17w18a";
+    public static String DiwModVersion = "112.1"; //Unsure if this has any use anymore
+    public static String Version = "1.11.2_b3"; //Updated to reflect a potential numbering system?
     public static double DiwModVersionNumeric = Double.parseDouble(DiwModVersion);
     public static String MC_SERVER_MOD_NAME = "Project Rails"; //ProjectRails: Change from Rainbow to ProjectRails
     public static String RainbowDataDirectory = "ProjectRails" + File.separator; //ProjectRails: Rainbow to ProjectRails
@@ -152,7 +152,7 @@ public class _DiwUtils {
     public static boolean UpdateNameColorOnTab = true;
     public static double netherDistanceRatio = 8.0D;
     public static long g_restartCountdown = -1L;
-    public static boolean heartbeat = true;
+    public static boolean heartbeat = true; //Does nothing yet
 
     public static MinecraftServer getMinecraftServer() {
         return minecraftServer;
-- 
2.10.1.windows.1

